CMakeLists.txt

```diff
@@ -127,8 +127,9 @@ target_sources(
 
 target_sources(obs-websocket PRIVATE deps/qr/cpp/QrCode.cpp deps/qr/cpp/QrCode.hpp)
 
-configure_file(src/plugin-macros.h.in src/plugin-macros.generated.h)
-target_sources(obs-websocket PRIVATE src/plugin-macros.generated.h)
+configure_file(src/plugin-macros.h.in plugin-macros.generated.h)
+target_sources(obs-websocket PRIVATE plugin-macros.generated.h)
+target_include_directories(obs-websocket PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Correct, but not necessary for 3.0 CMakeLists as `CMAKE_INCLUDE_CURRENT_DIR` is enabled.

Reply:
Remove the unnecessary `target_include_directories`.